rust-toolchain.toml

```diff
@@ -1,5 +1,5 @@
 [toolchain]
-channel = "stable"
+channel = "1.66.0"
```

Comment:
CI was failing because of the `wasm-opt` issue. This feels reasonable for now.

Reply:
Can we add a FIXME comment here and create an issue to track this? I'd like to get us back to stable as soon as we have a fix for this.

Reply:
Sorry for the back and forth here, but with [this patch](https://github.com/Shopify/javy/pull/217), this is no longer required.

Reply:
I know we use cargo-wasi, but we never trigger release builds as far as I know, so `wasm-opt` should never get invoked.